packages/gatsby-adapter-netlify/CHANGELOG.md

```diff
@@ -0,0 +1,78 @@
+- handle cases with large cached \_redirects and/or \_headers files [#38559](https://github.com/gatsbyjs/gatsby/issues/38559) ([db41d13](https://github.com/gatsbyjs/gatsby/commit/db41d1356c527cf4028142050978accd4abb1e9a))
+
+#### Chores
+
+- update dependency rimraf to ^5.0.5 [#38596](https://github.com/gatsbyjs/gatsby/issues/38596) ([37f2288](https://github.com/gatsbyjs/gatsby/commit/37f2288ee701d30d4d62ccb2f1d4487e7eb522b4))
+- added logging for cache handling [#38654](https://github.com/gatsbyjs/gatsby/issues/38654) ([f642940](https://github.com/gatsbyjs/gatsby/commit/f642940579fb2c96c22868b2b40f0983feaef087))
+
+### [1.0.4](https://github.com/gatsbyjs/gatsby/commits/gatsby-adapter-netlify@1.0.4/packages/gatsby-adapter-netlify) (2023-11-21)
+
+#### Features
+
+- support pathPrefix and trailingSlash options fix [#38666](https://github.com/gatsbyjs/gatsby/issues/38666) fix [#38701](https://github.com/gatsbyjs/gatsby/issues/38701) ([1090558](https://github.com/gatsbyjs/gatsby/commit/1090558010c5e73de49d5987ab38d30165581c51))
+
+### [1.0.3](https://github.com/gatsbyjs/gatsby/commits/gatsby-adapter-netlify@1.0.3/packages/gatsby-adapter-netlify) (2023-11-03)
+
+#### Bug Fixes
+
+- adapter use headerRoutes [#38652](https://github.com/gatsbyjs/gatsby/issues/38652) [#38674](https://github.com/gatsbyjs/gatsby/issues/38674) ([8ae5702](https://github.com/gatsbyjs/gatsby/commit/8ae5702433e8ab68b2742ef92f9816fbc84dcfd0))
+
+### [1.0.2](https://github.com/gatsbyjs/gatsby/commits/gatsby-adapter-netlify@1.0.2/packages/gatsby-adapter-netlify) (2023-10-26)
+
+#### Bug Fixes
+
+- respect 'force' and 'conditions' properties on redirects [#38657](https://github.com/gatsbyjs/gatsby/issues/38657) [#38664](https://github.com/gatsbyjs/gatsby/issues/38664) ([c43080f](https://github.com/gatsbyjs/gatsby/commit/c43080f49fe54022be141855db4e56d98398338c))
+
+#### Chores
+
+- added logging for cache handling [#38654](https://github.com/gatsbyjs/gatsby/issues/38654) [#38660](https://github.com/gatsbyjs/gatsby/issues/38660) ([4a76878](https://github.com/gatsbyjs/gatsby/commit/4a768788c91298317ff40388c288564efc52b763))
+
+### [1.0.1](https://github.com/gatsbyjs/gatsby/commits/gatsby-adapter-netlify@1.0.1/packages/gatsby-adapter-netlify) (2023-09-22)
+
+#### Bug Fixes
+
+- handle cases with large cached \_redirects and/or \_headers files [#38559](https://github.com/gatsbyjs/gatsby/issues/38559) [#38564](https://github.com/gatsbyjs/gatsby/issues/38564) ([56ddcce](https://github.com/gatsbyjs/gatsby/commit/56ddccecb8df4b04af956df9c83b7058d8008b26))
+
+## [1.0.0](https://github.com/gatsbyjs/gatsby/commits/gatsby-adapter-netlify@1.0.0/packages/gatsby-adapter-netlify) (2023-08-24)
+
+[🧾 Release notes](https://www.gatsbyjs.com/docs/reference/release-notes/v5.12)
+
+#### Features
+
+- Adapters [#38232](https://github.com/gatsbyjs/gatsby/issues/38232) ([b2d4aef](https://github.com/gatsbyjs/gatsby/commit/b2d4aefa7716e3ef02a19b530a8a7e4678d086e1))
+
+<a name="before-release-process"></a>
```

Comment:
Lack of above marker (and file in general) was causing changelog generation scripts to skip generating changelog for adapter:

https://github.com/gatsbyjs/gatsby/blob/222158e1ea88d0aedfbec10af1a24aae32acedb2/scripts/gatsby-changelog-generator/generate.js#L248-L255

https://github.com/gatsbyjs/gatsby/blob/222158e1ea88d0aedfbec10af1a24aae32acedb2/scripts/gatsby-changelog-generator/generate.js#L291